go.mod

```diff
@@ -75,7 +75,7 @@ require (
 	go.uber.org/atomic v1.11.0
 	go.uber.org/multierr v1.11.0
 	go.uber.org/zap v1.27.0
-	go.viam.com/api v0.1.383
+	go.viam.com/api v0.1.386-0.20250124195735-0d8303ff6186
```

Comment:
Cool how did you do this? There's an autogenerated pseudoversion just from PRs? Or from your branch?

Reply:
it was from a branch on the api repo, so might not work if on a fork